crates/wash-lib/src/parser/mod.rs

```diff
@@ -407,6 +407,10 @@ pub struct TinyGoConfig {
     pub tinygo_path: Option<PathBuf>,
     /// Whether to disable the `go generate` step in the build process. Defaults to false.
     pub disable_go_generate: bool,
+    /// The scheduler to use for the TinyGo build. Override the default scheduler (asyncify). Valid values are: none, tasks, asyncify.
```

Comment:
```suggestion
    /// The scheduler to use for the TinyGo build
    ///
    /// Override the default scheduler (asyncify). Valid values are: none, tasks, asyncify.
```

Leaving the space here gets you the summary line in the docsrs